### ISSUE TITLE
Membership Gaussmf

### ISSUE THREAD

**Issue by NetzerKathrin** [scikit-fuzzy/scikit-fuzzy#132]
Just something I wanted to remark:
The Gaussian membership function which I think should only get close towards 0 is actually getting 0.

**Comment by JDWarner** [comment]
@NetzerKathrin I can't reproduce this.  The operations are carried out floating-point, and even if I go out of my way to give the function integer inputs with a narrow sigma the result is still floating point.  This 

``` python
import numpy as np
import skfuzzy as fuzz

print(fuzz.gaussmf(np.asarray([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10], np.int16), 5, 1))
```
returns
``` python
[  1.38879439e-11   1.12535175e-07   1.23409804e-04   1.83156389e-02
   3.67879441e-01   1.00000000e+00   3.67879441e-01   1.83156389e-02
   1.23409804e-04   1.12535175e-07   1.38879439e-11]
```

Note that with a relatively narrow Gaussian (small sigma) these do get quite small rather quickly.   This behavior depends on NumPy, so it should be platform-independent.  

Perhaps your display settings for NumPy arrays round instead of displaying scientific notation?

**Comment by NetzerKathrin** [comment]
Thanks for your answer.
I got the exception that told me my antecedents would be too sparse.
After debugging it and looking into the terms of the antecedents I 
defined I noticed that at some point
the values actually turned zero (0.0). Perhaps you are right and it is 
because of some settings.
I worked around the exception by replacing these 0.0 by the smallest 
...e-... value in the terms[..].mf

Am 22.04.2017 um 13:37 schrieb Josh Warner:
